kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala

```diff
@@ -54,15 +54,15 @@ abstract class AbstractOperation(opType: OperationType, session: Session)
   protected def addTimeoutMonitor(queryTimeout: Long): Unit = {
     if (queryTimeout > 0) {
       val timeoutExecutor =
-        ThreadUtils.newDaemonSingleThreadScheduledExecutor("query-timeout-thread")
+        ThreadUtils.newDaemonSingleThreadScheduledExecutor("query-timeout-thread", false)
       val action: Runnable = () => cleanup(OperationState.TIMEOUT)
       timeoutExecutor.schedule(action, queryTimeout, TimeUnit.SECONDS)
       statementTimeoutCleaner = Some(timeoutExecutor)
     }
   }
 
   protected def shutdownTimeoutMonitor(): Unit = {
-    statementTimeoutCleaner.foreach(_.shutdown())
+    statementTimeoutCleaner.foreach(_.shutdown)
```

Comment:
It may be possible to keep the parentheses here, because the parentheses are omitted for a no-parameter method without side effects.